dist/stat_data.py

```diff
@@ -368,6 +368,8 @@ def __init__(self, name, desc, flags=''):
     CursorStat('cursor_modify_bytes', 'cursor modify key and value bytes affected', 'size'),
     CursorStat('cursor_modify_bytes_touch', 'cursor modify value bytes modified', 'size'),
     CursorStat('cursor_next', 'cursor next calls'),
+    CursorStat('cursor_next_hs_tombstone', 'cursor next calls that skip due to a globally visible history store tombstone'),
```

Comment:
Add `cursor_next_hs_tombstone` stat and separate usages of `cursor_next_hs_tombstone ` and `cursor_prev_hs_tombstone ` as per @bvpvamsikrishna 's request.